nucliadb_protos/resources.proto

```diff
@@ -472,6 +473,7 @@ message FieldFile {
     string url = 5;
     map<string, string> headers = 6;
     map<string, string> cookies = 7;
+    string css_selector = 8;
```

Comment:
I think this one is not necessary right? AFAIK we're only adding this to the link field

Reply:
right! fixed